components/support/locale/src/main/java/mozilla/components/support/locale/LocaleManager.kt

```diff
@@ -90,6 +95,26 @@ object LocaleManager {
         Storage.clear(context)
     }
 
+    /**
+     * Notifies the BrowserStore that there has been a [Locale] change.
+     *
+     * @param store The [BrowserStore]
+     * @param language The new [Locale] that has been chosen
+     */
+    internal fun notifyLocaleChanged(language: String) {
+        getBrowserStore().dispatch(UpdateLocaleAction(Locale(language)))
```

Comment:
Since the LocaleManager is a singleton object, what is the best way to access the BrowserStore?

Reply:
You could pass it as parameter :)
This way you wont have to keep a reference of it

Reply:
Since this action will be used in a few places I'll make a UseCase

Reply:
👍🏽